apps/web/src/features/resources/pages/conference-details.tsx

```diff
@@ -123,7 +123,53 @@ export const Conference = () => {
               <span className="text-newGray-4 mobile-body2 sm:desktop-h8">
                 {conference.location}, {conference.year}
               </span>
-              <div className="flex flex-wrap items-center gap-4 mt-4 lg:mt-8">
+              {(conference.twitterUrl || conference.websiteUrl) && (
```

Comment:
Lots of code duplication here, could we just adjust what's needed on mobile/desktop?